packages/react-composites/tests/browser/chat/ChatComposite.test.ts

```diff
@@ -1,31 +1,56 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT license.
-import puppeteer, { Browser, Page } from 'puppeteer';
-import { dataUiId } from '../utils';
+import path from 'path';
+import dotenv from 'dotenv';
+import { chromium, Browser, Page } from 'playwright';
+import faker from 'faker';
+import { dataUiId, createUserAndThread, encodeQueryData, getNameInitials } from '../utils';
 import { startServer, stopServer } from './app/server';
 import { CHAT_UI_IDS } from '../../../src';
-import { COMPONENT_UI_IDS } from '../../../../react-components';
+import { COMPONENT_UI_IDS } from '@internal/react-components';
+
+dotenv.config({ path: path.join(__dirname, '..', '.env') });
+
+jest.setTimeout(30000);
```

Comment:
For folks like me, can you add a commet here saying how many minutes / hours this is?

Reply:
What's the purpose of this? (Is it an arbitrary timeout to wait until the server etc has booted up? If so this both adds arbitrary extra time to tests and adds non-determinism where if the server takes longer our tests will fail)

Reply:
Heh, I assumed this was an overall timeout for the test suite in case something got stuck.

Yep: https://jestjs.io/docs/jest-object#jestsettimeouttimeout

And the docs say this is in milliseconds. So the timeout here is 30 seconds, up from the default of 5 seconds. My guess is that tstarting the server takes longer than 5 seconds, hence the call.

But all this should be in a comment above the `setTimeout` call.